source/application/SXRMB/SXRMB.h

```diff
@@ -19,7 +19,9 @@
 namespace SXRMB {
 
 	enum SXRMBErrorCodes {
-		ErrorSXRMBBeamlineShuttersTimeout = 290101  // 2-CLS 9-SXRMB 01 module-01 errorID-01
+		ErrorSXRMBBeamlineShuttersTimeout = 290101,  // 2-CLS 9-SXRMB 01 module-01 errorID-01
```

Comment:
I'm not entirely sold on this since we did not come up with a specific solution.  What are your thoughs @davidChevrier ?